### PR TITLE
rosdep: Add python django

### PR DESCRIPTION
**Pull request by tdenewiler**
Adding `python-django` and `python-django-extra-views` to `python.yaml`. Found packages for the following distributions where the generic name works for all OS versions I could find:

Arch:
  * https://www.archlinux.org/packages/?sort=&q=python-django&maintainer=&flagged=
  * Nothing for python-django-extra-views.

Debian:
  * https://packages.debian.org/search?keywords=python-django&searchon=names&suite=all&section=all
  * https://packages.debian.org/search?suite=all&section=all&arch=any&searchon=names&keywords=python-django-extra-views

Fedora:
  * https://apps.fedoraproject.org/packages/python-django
  * Nothing for python-django-extra-views.                                                                                  

Gentoo:
  * https://packages.gentoo.org/packages/dev-python/django
  * Nothing for python-django-extra-views.

Ubuntu:
  * https://packages.ubuntu.com/search?keywords=python-django&searchon=names&suite=all&section=all
  * https://packages.ubuntu.com/search?suite=all&section=all&arch=any&keywords=python-django-extra-views&searchon=names

I successfully tested all of those distributions with `rosdep resolve` using my forked repo/branch of rosdep as spelled out in the [Contributing rosdep rules](http://docs.ros.org/independent/api/rosdep/html/contributing_rules.html) documentation.

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -881,6 +881,15 @@ python-defusedxml:
     pip:
       packages: [defusedxml]
   ubuntu: [python-defusedxml]
+python-django:
+  arch: [python-django]
```
Comment (mikaelarguedas) [inline rosdep/python.yaml]: please use the Python 2 version: `python2-django` on arch

**Review by mikaelarguedas — COMMENTED** [review]
One nitpick about the python version on Arch, otherwise looks good to me.

Thanks @tdenewiler for the PR and for providing all the links :+1:

**Comment by tdenewiler** [comment]
@mikaelarguedas, thanks for the tip about arch.

**Review by mikaelarguedas — APPROVED** [review]
lgtm, thanks @tdenewiler for the PR and the package links :+1:

**Pull request merged**